web-client/src/components/UsaBanner.jsx

```diff
@@ -0,0 +1,81 @@
+import { connect } from '@cerebral/react';
+import { state, sequences } from 'cerebral';
+import React from 'react';
+
+import iconDotGov from '../../node_modules/uswds/dist/img/icon-dot-gov.svg';
+import iconHttps from '../../node_modules/uswds/dist/img/icon-https.svg';
+import usFlag from '../../node_modules/uswds/dist/img/favicons/favicon-57.png';
+
+/**
+ * Official US website banner
+ */
+export default connect(
+  {
+    showDetails: state.usaBanner.showDetails,
+    toggleUsaBannerDetails: sequences.toggleUsaBannerDetails,
+  },
+  function UsaBanner({ showDetails, toggleUsaBannerDetails }) {
+    return (
+      <section className="usa-banner">
+        <div className="usa-accordion">
+          <header className="usa-banner-header">
+            <div className="usa-grid usa-banner-inner">
+              <img src={usFlag} alt="U.S. flag" />
+              <p>An official website of the United States government&nbsp;</p>
+              <button
+                className="usa-accordion-button usa-banner-button"
+                aria-expanded={showDetails}
+                aria-controls="gov-banner"
+                onClick={() => toggleUsaBannerDetails()}
+              >
+                <span className="usa-banner-button-text">
+                  Here&apos;s how you know&nbsp;
```

Comment:
Whenever possible, I would prefer to see entities like `&apos;` nixed in favor of the actual character that's to be represented (`’`). The exceptions to this are those that HTML requires be recorded as entities (e.g., `&gt;`), plus `&nbsp;` since the Unicode character for a non-breaking space looks like any other space and can easily be overwritten accidentally. In our era of Unicode-everywhere, HTML character entities really just create technical debt.

```suggestion
                  Here’s how you know&nbsp;
```

Reply:
(This is not a thing that needs to be changed for this pull request to be accepted! It's jjust a guideline going forward.)

Reply:
I'd agree if this were raw HTML, though I think JSX is a bit more touchy. ESLint's recommended rules suggest escaping entities: https://github.com/yannickcr/eslint-plugin-react/blob/HEAD/docs/rules/no-unescaped-entities.md

Reply:
I believe that eslint rule only applies to straight single and double quotes, not the curled quote that @waldoj posted in his suggestion.  Since the curly quotes aren't part of a JSX string or an escape code, they're safe to use.